packages/react/src/components/Breadcrumb/Breadcrumb.js

```diff
@@ -25,13 +26,20 @@ const Breadcrumb = ({
   });
 
   return (
-    <nav className={className} {...rest}>
-      {children}
+    <nav aria-label={ariaLabel ? ariaLabel : 'Breadcrumb'}>
```

Comment:
if the aria label should be applied to `nav`, we should account for users that are directly setting the `aria-label` attribute since right now it looks like doing so would set `aria-label` on the `ol`

Reply:
How can we account for them previously setting `aria-label` via `...rest`?

Reply:
what does this look like ideally? do we have another component that we want to base this one?

Reply:
it would look like `<Breadcrumb aria-label="label">` using the standard hyphenated attribute rather than `<Breadcrumb ariaLabel="label">`. you can see the standard attribute being targeted in the [`AriaLabelPropType` helper](https://github.com/carbon-design-system/carbon/blob/master/packages/react/src/prop-types/AriaPropTypes.js)